content/doc/applications/php/_index.md

```diff
@@ -106,12 +106,20 @@ date.timezone=Europe/Paris
 
 ##### Header injection
 
-Usually, you can use an `.htaccess` file to create / update / delete headers.
-You won't be able to do it from the `.htaccess` file if the headers come from a `.php` file, because `php-fpm` ignores the `mod_headers` plugin.
-It works fine for static files directly served by apache.
+###### With .htaccess
 
-So if you need to inject headers on HTTP responses (for instance for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS)),
-you have to do it from PHP (you can't do it from `.htaccess`).
+You can use a `.htaccess` file to create or update headers, but you can't delete them.
+
+If you need to inject headers on HTTP responses (for instance for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS)),
+you can do it like this :
+
+```sh
+Header set Access-Control-Allow-Origin "*"
+```
```

Comment:
Maybe better to use a callout here: 

```suggestion
To inject headers on HTTP responses, add this configuration to `.htaccess` file:

```sh
Header set Access-Control-Allow-Origin "*"
```

{{< callout type="info" >}}
You can use a `.htaccess` file to create or update headers, but you not delete them.
{{< /callout >}}
```